sdk/synapse/test-resources.json

```diff
@@ -0,0 +1,403 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "baseName": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().name]",
+            "metadata": {
+                "description": "The base resource name."
+            }
+        },
+        "tenantId": {
+            "type": "string"
+        },
+        "testApplicationId": {
+            "type": "string"
+        },
+        "testApplicationSecret": {
+            "type": "string"
+        },
+        "workspacePrefix": {
+            "type": "String",
+            "defaultValue": "workspace"
+        },
+        "location": {
+            "type": "String",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "The location of the resource. By default, this is the same as the resource group."
+            }
+        },
+        "defaultDataLakeStorageAccountNamePrefix": {
+            "type": "String",
+            "defaultValue": "storage"
+        },
+        "defaultDataLakeStorageFilesystemNamePrefix": {
+            "type": "String",
+            "defaultValue": "filesystem"
+        },
+        "sqlAdministratorLogin": {
+            "type": "String",
+            "defaultValue": "sqladminuser"
+        },
+        "sqlAdministratorLoginPassword": {
+            "defaultValue": "",
+            "type": "SecureString"
+        },
+        "setWorkspaceIdentityRbacOnStorageAccount": {
```

Comment:
This template has a lot of parameters. Do we have a scenario where we would need to change them or is it a copy from somewhere else?

Reply:
Yes, this template is downloaded from Azure portal. When deploy a synapse workspace, it uses these parameters.